### DISCUSSION TITLE
Tongou circuit breaker QCB2 not connecting (Sorry, there is no support for this device)

### DISCUSSION THREAD

**Discussion by pbogaert** [make-all/tuya-local#2112] [Q&A] [answered]
Hi All,

I'm a noob in this stuff but at least I'm trying. 

I looked at the Tuya-local integration and installed it when I saw the QCB2 in de [DEVICES.md](https://github.com/make-all/tuya-local/blob/main/DEVICES.md) list. However, following all the steps I end up with "Sorry, there is no support for this device"

Is there anything I'm doing wrong?

steps taken:

1. installed tuya-local via HACS
2. added integration 
3. selected "Smart Life cloud-assisted device setup." (also tried the manual way without success)
4. the next steps it reached for the correct IP and when I pressed send, it took about 45 seconds to come back with "Sorry, there is no support for this device"

So I'm at a dead end.

Pleas help as I don'y like the cloud integration.

cheers,
![Scherm­afbeelding 2024-07-14 om 14 54 38](https://github.com/user-attachments/assets/9228c028-3e97-4787-a9ac-b548bd46c383)

![Scherm­afbeelding 2024-07-14 om 15 08 28](https://github.com/user-attachments/assets/2bcad48a-d1f0-4ddf-a172-a15b332d728f)

Paul

**Comment by make-all** [comment]
There seem to be several versions of this device. One more has just been added and not yet released.

**Reply by pbogaert** [reply]
Thanks Jason, if you need anything from my device, please let me know.

cheers,

Paul

**Reply by pbogaert** [reply, marked as answer]
That's funny, I shout you a coffee and all of a sudden after trying my luck again, it started working!!!!

Many thanks Jason!
